docs/pages/eas-update/estimate-bandwidth.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Estimate bandwidth usage
+description: Learn how to estimate bandwidth usage with EAS Update.
+---
+
+# Understanding Update Bandwidth Usage
+
+EAS Update enables an app to update its own non-native pieces (such as JS, styling, and images) over-the-air. This guide explains how bandwidth is consumed and optimized.
+
+## Bandwidth Calculation Breakdown
+
+Each billing cycle includes a predefined bandwidth allocation, which translates to a specific number of updates your users can download. Here's how we estimate bandwidth usage:
```

Comment:
Each subscription plan includes bandwidth per cycle, the cycles themselves don't include bandwidth.